src/test/java/it/pagopa/pn/bff/pnclient/userattributes/PnUserAttributesClientImplTest.java

```diff
@@ -0,0 +1,99 @@
+package it.pagopa.pn.bff.pnclient.userattributes;
+
+import it.pagopa.pn.bff.exceptions.PnBffException;
+import it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.api.ConsentsApi;
+import it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.model.Consent;
+import it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.model.ConsentAction;
+import it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.model.ConsentType;
+import it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.model.CxTypeAuthFleet;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@ContextConfiguration(classes = {PnUserAttributesClientImpl.class})
+@ExtendWith(SpringExtension.class)
+class PnUserAttributesClientImplTest {
+    @Autowired
+    private PnUserAttributesClientImpl pnUserAttributesClient;
+    @MockBean(name = "it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.api.ConsentsApi")
+    private ConsentsApi consentsApi;
+
+    @Test
+    void getTosConsentTest() throws RestClientException {
+        when(consentsApi.getConsentByType(
+                Mockito.<String>any(),
```

Comment:
Use here and in all tests the Mockito.anyString(), Mockito.any(CxTypeAuthFleet.class) ecc...